db/migrate/20210620222418_create_offer_responses.rb

```diff
@@ -0,0 +1,13 @@
+class CreateOfferResponses < ActiveRecord::Migration[6.1]
+  def change
+    create_table :offer_responses do |t|
+      t.integer   :user_id
+      t.integer   :offer_id
+
+      t.timestamps
+    end
+
+  add_foreign_key :offer_responses, :users, column: :user_id
+  add_foreign_key :offer_responses, :offers, column: :offer_id
```

Comment:
would be good to add 2 indexes for offer_id and user_id also

Reply:
@shreyas098 we probably need a unique composite index to avoid duplicates. And we need tests for duplicates generally